src/main/java/net/sf/rails/game/financial/StockRound.java

```diff
@@ -263,7 +263,7 @@ public void setBuyableCerts() {
 
                 if (!cert.isPresidentShare()) {
                     price = comp.getIPOPrice() / unitsForPrice;
-                    if (price <= playerCash) {
+                    if ((price*shares) <= playerCash) {
```

Comment:
Within this block, the "shares" variable is only referenced here, which makes creating the variable unnecessary.

It'd be simpler for this to be "price*cert.getShares()"